### PR TITLE
LPS-165366 and LPS-165948 [EVP] Create Request Status functionality, Adding some Request Form Configuration, Setting Organization Form Status funcionality

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by juliocezarjc** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/request-form-handler/index.js [hunk 1]
```diff
@@ -10,8 +10,20 @@
...
+const grantAmount = document.querySelector('[name="grantAmount"]');
+const managerEmailAddress = document.querySelector(
+	'[name="managerEmailAddress"]'
+);
+const totalHoursRequested = document.querySelector(
+	'[name="totalHoursRequested"]'
+);
+const startDate = document.querySelector('[name="startDate"]');
+const endDate = document.querySelector('[name="endDate"]');
+
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/request-form-handler/index.js:26]: Please, use ```fragmentElement``` instead ```document```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/organization-form-handler/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const requestStatus = document.querySelector('[name="organizationStatus"]');
```
[the remark this answers is not in the document]
Comment (JoMaAlves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/organization-form-handler/index.js:12]: As we are accessing elements outside the fragment, we need to use `document` in this case.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/request-form-handler/index.js (hunk 1 above)
Comment (JoMaAlves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/request-form-handler/index.js:26]: As we are accessing elements outside the fragment, we need to use `document` in this case.

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by juliocezarjc** [comment]
Just starting reviewing :)

**Review by juliocezarjc — APPROVED** [review]
Nice job! 🤙

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**